plugins/woocommerce/src/Utilities/DBUtil.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * A class of utilities for dealing with Database management.
+ */
+
+namespace Automattic\WooCommerce\Utilities;
+
+use Automattic\Jetpack\Constants;
+
+class DBUtil {
```

Comment:
Given that the methods in this class are not idempotent (since they depend on the database state), I think it should have non-static methods and be instantiated via dependency injection container. This will ease the task of writing unit tests, as we'll be able to easily mock the database behavior.

Also I know I suggested the `DBUtil` name myself, but maybe a more explicit name like `DatabaseUtil` would be more appropriate.